### PR TITLE
Submits to Code Climate

### PR DESCRIPTION
**Pull request by zgosalvez**
**Known issue:**

```log
% CODECLIMATE_DEBUG=1 codeclimate analyze --dev
...truncated...
I, [2020-10-31T04:04:47.889734 #1]  INFO -- : starting engine dart
D, [2020-10-31T04:04:47.890442 #1] DEBUG -- : /config.json content: {"enabled"=>true, "channel"=>"stable", "include_paths"=>[".DS_Store", ".codeclimate.yml", ".dart_tool/", ".flutter-plugins", ".flutter-plugins-dependencies", ".github/", ".gitignore", ".idea/", ".metadata", ".packages", ".vscode/", ".whitesource", "analysis_options.yaml", "android/", "app.iml", "assets/", "build/", "coverage/", "ios/", "lib/", "pubspec.lock", "pubspec.yaml"], "debug"=>"1"}
D, [2020-10-31T04:04:47.892429 #1] DEBUG -- : docker run: ["docker", "run", "--name", "cc-engines-dart-stable-ff1a5ce6-06ac-480d-b0b9-53720f00b4db", "--cap-drop", "all", "--label", "com.codeclimate.label=28212475-1198-47c8-a6ca-1076205b89d1", "--log-driver", "none", "--memory-swap", "-1", "--net", "none", "--rm", "--volume", "/Users/redacted/project:/code:ro", "--volume", "/tmp/cc/aa2567d6-43d4-4d17-9083-e14ae13fdde1:/config.json:ro", "--user", "9000:9000", "codeclimate/codeclimate-dart"]
D, [2020-10-31T04:04:50.214887 #1] DEBUG -- : engine stdout: [...redacted...]
#<Thread:0x000055d3b8d49860@/usr/src/app/lib/cc/analyzer/container.rb:120 run> terminated with exception (report_on_exception is true):
Traceback (most recent call last):
        8: from /usr/src/app/lib/cc/analyzer/container.rb:121:in `block in read_stdout'
        7: from /usr/src/app/lib/cc/analyzer/container.rb:121:in `each_line'
        6: from /usr/src/app/lib/cc/analyzer/container.rb:125:in `block (2 levels) in read_stdout'
        5: from /usr/src/app/lib/cc/analyzer/engine.rb:41:in `block in run'
        4: from /usr/src/app/lib/cc/analyzer/engine.rb:59:in `handle_output'
        3: from /usr/src/app/lib/cc/analyzer/engine_output_filter.rb:11:in `filter?'
        2: from /usr/src/app/lib/cc/analyzer/engine_output.rb:12:in `issue?'
        1: from /usr/src/app/lib/cc/analyzer/engine_output.rb:55:in `valid_with_type?'
/usr/src/app/lib/cc/analyzer/engine_output.rb:55:in `[]': no implicit conversion of String into Integer (TypeError)
error: (TypeError) no implicit conversion of String into Integer
D, [2020-10-31T04:04:50.280056 #1] DEBUG -- : backtrace: /usr/src/app/lib/cc/analyzer/engine_output.rb:55:in `[]'
        /usr/src/app/lib/cc/analyzer/engine_output.rb:55:in `valid_with_type?'
        /usr/src/app/lib/cc/analyzer/engine_output.rb:12:in `issue?'
        /usr/src/app/lib/cc/analyzer/engine_output_filter.rb:11:in `filter?'
        /usr/src/app/lib/cc/analyzer/engine.rb:59:in `handle_output'
        /usr/src/app/lib/cc/analyzer/engine.rb:41:in `block in run'
        /usr/src/app/lib/cc/analyzer/container.rb:125:in `block (2 levels) in read_stdout'
        /usr/src/app/lib/cc/analyzer/container.rb:121:in `each_line'
        /usr/src/app/lib/cc/analyzer/container.rb:121:in `block in read_stdout'
```

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,22 +1,27 @@
...
 RUN pub get
 RUN dart2native /app/bin/metrics.dart -o /app/metrics
 
+# Install engine:
+
 FROM bitnami/minideb
-RUN adduser -u 9000 --disabled-password app
-WORKDIR /app
-COPY engine.json ./
+LABEL maintainer="Zennon Gosalvez <1798166+zgosalvez@users.noreply.github.com>"
```
Comment (sinegovsky-ivan) [inline Dockerfile]: Please don't change the maintainer

File: engine.json [hunk 2]
```diff
@@ -2,8 +2,8 @@
   "name": "dart",
   "description": "Code Climate Dart-Engine",
   "maintainer": {
-    "name": "Sinegovsky Ivan",
-    "email": "ivan.sinegovsky@team.wrike.com"
+    "name": "Zennon Gosalvez",
```
Comment (sinegovsky-ivan) [inline engine.json]: please revert

**Review by sinegovsky-ivan — APPROVED** [review]

**Comment by noelia-lencina** [comment]
Hi, @zgosalvez - I wanted to let you know that we have added this plugin to the official CLI, so now you can run `codeclimate analyze` and configure the plugin in your `.codeclimate.yml` file (no need to use `--dev` and build the image locally). Thanks for your contribution!

**Comment by efueger** [comment]
Thanks @noelia-lencina ! Also, @zgosalvez, you can: 
- edit this [draft help doc](https://docs.codeclimate.com/docs/dart-code-metrics)
- add dart to this [table](https://docs.codeclimate.com/docs/list-of-engines#community-supported-plugins)

You'll find the edit option here:
image

Let me know if you have any questions and thanks for your help! 💪

**Pull request merged**